clean_build.bat

```diff
@@ -10,6 +10,18 @@ set B_BONJOUR=C:\Program Files\Bonjour SDK
 set savedir=%cd%
 cd /d %~dp0
 
+REM cmake generator name for the target build system
+if "%VisualStudioVersion%"=="15.0" (
+    set cmake_gen=Visual Studio 15 2017 Win64
+) else if "%VisualStudioVersion%"=="16.0" (
+    set cmake_gen=Visual Studio 16 2019

```

Comment:
Is `Win64` no longer needed?

Reply:
I believe so.

Reply:
Barrier doesn't appear to be buildable for Win32. If that's the case, I can normalize the generator strings and specify x64 separately.

Reply:
Yes, 32-bit build is not currently supported.